prisma/schema.sql

```diff
@@ -0,0 +1,170 @@
+CREATE TYPE provinces AS ENUM('NL', 'PE', 'NS', 'NB', 'QC', 'ON', 'MB', 'SK', 'AB', 'BC', 'YT', 'NT', 'NU');
+CREATE TYPE weekdays AS ENUM ('MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT', 'SUN');
+
+-- Create users table
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY NOT NULL,
+  name VARCHAR(255),
+  email VARCHAR(255) UNIQUE,
+  email_verified TIMESTAMPTZ,
+  image TEXT,
+  created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+-- Create email verification requests table
+CREATE TABLE verification_requests (
+  id SERIAL PRIMARY KEY NOT NULL,
+  identifier VARCHAR(255) NOT NULL,
+  token VARCHAR(255) NOT NULL UNIQUE,
+  expires TIMESTAMPTZ NOT NULL,
+  created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+-- create program table
+CREATE TABLE program (
+  program_id SERIAL PRIMARY KEY NOT NULL,
+  price INTEGER NOT NULL, -- cents
+  start_date TIMESTAMPTZ NOT NULL,
+  end_date TIMESTAMPTZ NOT NULL,
+  weekday weekdays NOT NULL,
+  start_time_minutes INTEGER NOT NULL,
+  duration_minutes INTEGER NOT NULL,
+  space_total INTEGER NOT NULL,
+  space_available INTEGER NOT NULL,  
+  volunteer_space_total INTEGER NOT NULL,
+  volunteer_space_available INTEGER NOT NULL,
+  is_archived BOOLEAN DEFAULT false NOT NULL
+);
+
+-- create parent table
+CREATE TABLE parent (
+  parent_id SERIAL PRIMARY KEY NOT NULL,
+  first_name VARCHAR(255) NOT NULL,
+  last_name VARCHAR(255) NOT NULL,
+  phone_number VARCHAR(50) NOT NULL,
+  email VARCHAR(255) UNIQUE NOT NULL,
+  is_low_income BOOLEAN DEFAULT false,
+  address_line1 VARCHAR(255) NOT NULL,
+  address_line2 VARCHAR(255),
+  postal_code VARCHAR(10) NOT NULL,
+  city_name VARCHAR(50) NOT NULL,
+  province provinces NOT NULL,
+  preferred_language VARCHAR(20) NOT NULL
```

Comment:
can this be turned into an enum? we have a handful of languages we're planning to support, right?